app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -484,7 +485,7 @@ def parse_args(args)
 
       # see #26402 - consume scsi_controller_type from hammer as a default scsi type
       scsi_type = args.delete(:scsi_controller_type)
```

Comment:
We will need to create a generic option for Hammer too here: instead of `scsi_controller_type` it would become `controller_type`. Plus we will need to deprecate the older `scsi_controller_type`

Reply:
scsi_controller_type has been removed from compute attributes in this [commit](https://github.com/theforeman/hammer-cli-foreman/commit/7b59a148f508e314b1817930a94b0c989e803c35) Its best we remove this part all together as we use `scsi_controller` attribute to pass the type and the key 



![Screenshot from 2024-05-30 15-46-54](https://github.com/theforeman/foreman/assets/115526987/2d2d2e0c-a7b9-49a2-b09f-725b6a8d241e)